src/www/ui/template/advice_license-edit.html.twig

```diff
@@ -12,6 +12,24 @@
   <input type="hidden" name="mod" value="advice_license"/>
   <input type="hidden" name="rf" value="{{ rf_pk }}"/>
   <table>
+  {% if rf_lastmodified is not empty %}
+    <tr>
+      <td>{{ 'Username Created'|trans }}:</td>
+      <td><span type="input" name="useridcreated"> {{ rf_user_fk_created|e }}</span></td>
+    </tr>    
+    <tr>
+      <td>{{ 'Date of Creation'|trans }}:</td>
+      <td><span type="input" name="creationdate"> {{ rf_creationdate|e }}</span></td>
+    </tr>
+    <tr>
+      <td>{{ 'Username Modified'|trans }}:</td>
```

Comment:
If you are changing above, please consider "User Modified" here as well.

Reply:
yeah sure